internal/common/gamestate/history.go

```diff
@@ -1,21 +1,21 @@
-package iigointernal
+package gamestate
 
 import (
 	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
 	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 )
 
 type Accountability struct {
-	clientID shared.ClientID
-	pairs    []rules.VariableValuePair
+	ClientID shared.ClientID
+	Pairs    []rules.VariableValuePair
 }
 
 var TurnHistory []Accountability
```

Comment:
![image](https://user-images.githubusercontent.com/33488131/103108252-04f4c980-463e-11eb-9292-c0107465999d.png)

Reply:
Not sure what to do here, cos we need it exported for other packages to read. I'll try moving to shared, but it'll still need to be exported ;(